src/conn/conn_api.c

```diff
@@ -1994,9 +1994,13 @@ __wt_timing_stress_config(WT_SESSION_IMPL *session, const char *cfg[])
 {
 	static const WT_NAME_FLAG stress_types[] = {
 		{ "checkpoint_slow",	WT_TIMING_STRESS_CHECKPOINT_SLOW },
```

Comment:
I think splitting the `split_race` conditions up is a great idea - it should make this mode more effective at uncovering problems. I'd like for that to be documented somewhere though - maybe in a comment here?

Reply:
@agorrod 
> I'd like for that to be documented somewhere though - maybe in a comment here?

I tried to add a comment in 54dddf7, but not entirely sure what you wanted the comment to say. Feel free to just reword to match what you're looking for.